src/server/search-index/metrics-images.search-index.ts

```diff
@@ -306,7 +309,10 @@ export const imagesMetricsDetailsSearchIndex = createSearchIndexUpdateProcessor(
         ) AS "hasMeta",
         (
           CASE
-            WHEN i.meta->>'civitaiResources' IS NOT NULL
+            WHEN i.meta->>'civitaiResources' IS NOT NULL 
+              OR i.meta->>'workflow' IS NOT NULL AND i.meta->>'workflow' = ANY(ARRAY[
+                ${Prisma.join(workflows)}
```

Comment:
We have to make sure workflows is not and empty array, otherwise `Prisma.join()` throws an error